src/main/java/org/springframework/data/elasticsearch/core/ElasticsearchRestTemplate.java

```diff
@@ -764,6 +766,32 @@ public void bulkUpdate(List<UpdateQuery> queries) {
 		}
 	}
 
+	private void setBulkOptions(BulkRequest bulkRequest, BulkOptions bulkOptions) {
+		if (bulkOptions != null) {
```

Comment:
Should we reduce the scope and do an earlier return if `bulkOptions` is `null`?

Reply:
changed it, and added `@Nullable` annotations that I missed before